logic/atrium-logic-common/src/main/kotlin/ch/tutteli/atrium/logic/creating/iterable/contains/creators/impl/InAnyOrderEntriesAssertionCreator.kt

```diff
@@ -92,26 +123,23 @@ class InAnyOrderEntriesAssertionCreator<E : Any, T : IterableLike>(
         return group to count
     }
 
-    //TODO 0.18.0 check if this is still state of the art to add a hint that no assertion was created
```

Comment:
please re-add the TODO, it's something I still need to check in the next version